frontend-project/src/pages/letters/documentTypes/locales/en-US.ts

```diff
@@ -0,0 +1,6 @@
+export default {
+  'documentTypes-list.page-header-content': 'Document types list',
```

Comment:
Document type list (?)
jest może jakiś anglista na sali?

Reply:
zmienione na 'List of document types'